### PR TITLE
SLING-11036: use dedicated threadpool for scheduled jobs

### PR DESCRIPTION
**Pull request by Buuhuu**

### CODE REVIEW

File: src/main/java/org/apache/sling/sitemap/impl/SitemapStorage.java [hunk 1]
```diff
@@ -323,8 +325,15 @@ public String writeSitemap(@NotNull Resource sitemapRoot, @NotNull String name,
     public Collection<SitemapStorageInfo> getSitemaps(Resource sitemapRoot, Collection<String> names) {
         try (ResourceResolver resolver = resourceResolverFactory.getServiceResourceResolver(AUTH)) {
             Resource topLevelSitemapRoot = getTopLevelSitemapRoot(sitemapRoot);
-            Predicate<SitemapStorageInfo> filter;
+            String storagePath = rootPath + topLevelSitemapRoot.getPath();
+            Resource storageResource = resolver.getResource(storagePath);
 
+            if (storageResource == null) {
+                LOG.debug("Resource at {} does not exist.", storagePath);
+                return Collections.emptySet();
```
Comment (joerghoh) [inline src/main/java/org/apache/sling/sitemap/impl/SitemapStorage.java:333]: This change does not look related to the topic of the PR. Can you create a dedicated Issue and PR which contains this change?

**Review by joerghoh — CHANGES_REQUESTED** [review]

File: src/main/java/org/apache/sling/sitemap/impl/SitemapStorage.java (hunk 1 above)
Comment (Buuhuu) [reply src/main/java/org/apache/sling/sitemap/impl/SitemapStorage.java:333]: Right, that was not related. Let me revert this.
